server/src/rest_service.py

```diff
@@ -1,7 +1,37 @@
 from config import config
 import requests
+from schema import User
+
 
 def slack_users():
     token_data = {'token': config['slackToken']}
     return requests.get(config['slackApiUrl'] + '/users.list', params=token_data)
 
+
+def app_users(session):
```

Comment:
This is more knit picking, but would it be more clear if we called this variable db_session instead of just session? The first time I was reading through I thought it was referencing web sessions.